Project/src/main/java/com/github/lgooddatepicker/components/CalendarPanel.java

```diff
@@ -964,13 +965,16 @@ private void labelIndicatorSetColorsToDefaultState(JLabel label) {
         }
         if (label == labelMonth || label == labelYear) {
             label.setBackground(settings.getColor(DateArea.BackgroundMonthAndYearMenuLabels));
+            label.setForeground(settings.getColor(DateArea.TextMonthAndYearMenuLabels));
             monthAndYearInnerPanel.setBackground(settings.getColor(DateArea.BackgroundMonthAndYearMenuLabels));
         }
         if (label == labelSetDateToToday) {
             label.setBackground(settings.getColor(DateArea.BackgroundTodayLabel));
+            label.setForeground(settings.getColor(DateArea.TextTodayLabel));
         }
         if (label == labelClearDate) {
             label.setBackground(settings.getColor(DateArea.BackgroundClearLabel));
+            label.setForeground(settings.getColor(DateArea.TextClearLabel));
```

Comment:
Same question as above, the Background is customizeable but you are setting the *Foreground* here :thinking:

Reply:
Please disregard my precious comment. `setForground` means setting the text color in this context, which is needed here to restore the color after highlighting is finished.